src/containers/Contact/ContactContainer.jsx

```diff
@@ -1,7 +1,13 @@
 import React from 'react';
 
+import ContactList from '../../components/Contact/ContactList';
```

Comment:
ini bisa dipanggil seperti ini
```suggestion
import ContactList from 'components/Contact/ContactList';
```